src/main/java/org/folio/qm/controller/ErrorHandling.java

```diff
@@ -49,7 +50,17 @@ public Error handleNotFoundException(NotFoundException e, HttpServletResponse re
 
   @ExceptionHandler(MissingServletRequestParameterException.class)
   public Error handleMissingParameterException(MissingServletRequestParameterException e, HttpServletResponse response) {
-    var message = e.getParameterName() + " is required";
+    var message = "Parameter '" + e.getParameterName() + "' is required";
+    return buildBadRequestResponse(response, message);
+  }
+
+  @ExceptionHandler(MethodArgumentTypeMismatchException.class)
```

Comment:
nice to have well defined exceptions and ability to easily customize their handling.. Spring rocks!